addons/survey/static/src/scss/survey_templates_results.scss

```diff
@@ -41,7 +41,7 @@
         .o_survey_results_question_wrapper:has(button.collapsed) {
             display: none;
         }
-        .o_survey_user_responses_table_wrapper {
+        .o_survey_results_table_wrapper {
```

Comment:
Hi :slightly_smiling_face:
I renamed it for multiple reasons:
- it is smaller than the previous one
- every classes of the results page usually start with `o_survey_results_...`
- the class is now applied on the parent of the table instead of on the table directly. So using `table_wrapper` without the notion of `user_responses` seems more accurate to me.